projects/abtesting/src/app/core/auth/auth.service.ts

```diff
@@ -38,4 +42,41 @@ export class AuthService {
   setRedirectionUrl(redirectUrl: string) {
     this.store$.dispatch(AuthActions.actionSetRedirectUrl({ redirectUrl }));
   }
+
+  setUserPermissions(role: UserRole) {
+    switch (role) {
+      case UserRole.ADMIN:
+        this.userPermissions$.next({
+          experiments: { create: true, read: true, update: true, delete: true },
+          users: { create: true, read: true, update: true, delete: true },
+          logs: { create: true, read: true, update: true, delete: true },
+          manageRoles: { create: true, read: true, update: true, delete: true }
+        });
+        break;
+      case UserRole.CREATOR:
+        this.userPermissions$.next({
+          experiments: { create: true, read: true, update: true, delete: true },
+          users: { create: true, read: true, update: true, delete: true },
+          logs: { create: false, read: true, update: false, delete: false },
+          manageRoles: { create: false, read: false, update: false, delete: false }
+        });
+        break;
+      case UserRole.MANAGER:
```

Comment:
Bro can you update this name to `User Manager`. I think Manager is not appropriate to the kind of permissions assigned

Reply:
Updated